sass/styles.scss

```diff
@@ -0,0 +1,505 @@
+$font-sans:'Open Sans',
+sans-serif;
+$blue:#0240f4;
+$square-cell:minmax(100px, min(300px, 33vw));
+$square-cell2:minmax(100px, min(300px, 16vw));
+
+* {
+    box-sizing: border-box;
+}
+
+//gneral text
+
+body {
+    background-color: darkgray;
+    color: $blue;
+    font-family: $font-sans;
+    font-size: 14px;
+    box-shadow: 30px 50px 10px grey;
+    margin: 0;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+h1 {
+    font-size: 160px;
+    line-height: 1.3;
+    margin: 0;
+}
+
+h2 {
+    align-items: baseline;
+    font-size: 160px;
+    line-height: 0.8;
+    margin: 0;
+}
+
+h3 {
+    margin: 0 0 0 5px;
+}
+
+main {
+    background-color: white;
+    display: grid;
+    grid-template-columns: repeat(3, $square-cell);
+    grid-template-rows: repeat(16, $square-cell);
+}
+
+header,
+footer {
+    background-color: white;
+    display: grid;
+    grid-template-columns: repeat(3, $square-cell);
+    grid-template-rows: auto;
+    box-shadow: 30px 50px 10px grey;
+}
+
+ul {
+    list-style-type: none;
+    padding-left: 5px;
+    margin: 0;
+}
+
+.twentynineteen {
+    letter-spacing: -15px;
+    line-height: 195px;
+}
+
+.ten,
+.eleven,
+.twelve,
+.sixteen,
+.seventeen,
+.eighteen {
+    letter-spacing: -25px;
+    margin-left: -7px;
+}
+
+.i1,
+.i4 {
+    display: flex;
+    align-items: baseline;
+}
+
+//graphics
+
+.circle {
+    border-radius: 200px;
+}
+
+.circlesquare {
+    border-radius: 200px 0 0 200px;
+}
+
+.circle,
+.circlesquare,
+.square {
+    background-color: $blue;
+}
+
+.circle,
+.circlesquare,
+.square {
+    &:hover {
+        background-color: white;
+    }
+}
+
+//grids
+
+.a1,
+.a3 {
```

Comment:
I expect this naming strategy for your grid elements worked well when it was in one view (desktop or mobile), but then maybe broke down when you tried to do the responsive styles. Keep this in mind as you think about how to name classes in the future. 👍🏻